### ISSUE TITLE
why you dont have an examples for each gif?

### ISSUE THREAD

**Issue by Haimma** [TaumuLu/react-scroll-paged-view#18]
why you dont have an examples for each gif? 

can you publish please

**Comment by TaumuLu** [comment]
> why you dont have an examples for each gif?
> 
> can you publish please

Yes, I am currently switching to TS, all examples will be maintained with a separate project
